src/optimized_algorithms/main.cpp

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Let's rename this to a different "main" file, as I suspect we'll need to start writing multiple "test" things anyway.

Maybe "main2.cpp"? "mainsowmya.cpp"? Just something distinct. Then I'll make some "test" directories to put them in.